src/map/utils/trustutils.cpp

```diff
@@ -343,13 +343,13 @@ void LoadTrustStatsAndSkills(CTrustEntity* PTrust)
     uint8 jobHPGrade = grade::GetJobGrade(mJob, 0);
     uint8 jobMPGrade = grade::GetJobGrade(mJob, 1);
 
-    uint8 sjobHPGrade = grade::GetJobGrade(sJob, 0);
+    //uint8 sjobHPGrade = grade::GetJobGrade(sJob, 0);
```

Comment:
ugh. I thought this was refactored out. They should be built from mob family system. No wonder you've had issues they are still being built like players and that's never going to align with retail. This was a part of my originally holding back the merge on DSP.

Reply:
Yeah, this is the _last_ set of tweaks before I rewrite everything properly. I just wanted to bring trusts down to reasonable and non-facerolly levels of hp/mp